### PR TITLE
Transform the accordion reviews component into a partial

### PR DESCRIPTION
**Pull request by danidoni**
We move some logic into BsRequest because we're asking too much information to it and to its reviews. That could live there and just tell request to do it for us.

The component per se, has almost no logic and we're not reusing it anywhere else.

### CODE REVIEW

**Review by krauselukas — COMMENTED** [review]
@danidoni you forgot to delete the `html.haml` file of the accordion component. There is also the `accordion-reviews-component.scss` stylesheet. Maybe it should also be renamed.

**Comment by danidoni** [comment]
> @danidoni you forgot to delete the `html.haml` file of the accordion component. There is also the `accordion-reviews-component.scss` stylesheet. Maybe it should also be renamed.

Deleted the scss file. Good catch!

> [EDIT]: sorry, I just couldn't read the diff, it didn't showed up as a delete, but as moved to the partial. so all good, I would just rename the scss file.

Yeah, I don't understand why does that sometimes...

**Review by krauselukas — CHANGES_REQUESTED** [review]
image

**Comment by krauselukas** [comment]
@danidoni are you sure the css code can be removed? is it not used in the partial?

**Comment by danidoni** [comment]
> @danidoni are you sure the css code can be removed? is it not used in the partial? my review was more about the naming of the file :)

My bad :see_no_evil: Fixed

File: src/api/app/assets/stylesheets/webui/accordion-reviews-component.scss
Comment (krauselukas) [inline src/api/app/assets/stylesheets/webui/accordion-reviews-component.scss:1]: @danidoni this cannot be dropped, the css code is still used. I would just rename it, or move the code somewhere.

Comment (danidoni) [reply src/api/app/assets/stylesheets/webui/accordion-reviews-component.scss:1]: Alright, I added the scss file again. Now it looks like before.

**Review by krauselukas — APPROVED** [review]

**Pull request merged**